mflowgen/pad_frame/custom-init/outputs/io_fillers.tcl

```diff
@@ -0,0 +1,94 @@
+    # Note: Before using addIoFiller, run the globalNetCommand to provide
+    # global-net-connection rules for supply pins of the added
+    # fillers. Without these rules, the built-in design-rule checks of
+    # addIoFiller will not be accurate.
+    #
+    # (The globalNetCommand note does not appear in the stylus version of the 
+    # man page (add_io_fillers)
+
+    # [stevo]: add -logic so fillers get RTE signal connection
+    #   [steveri 11/2019]: Dunno where "-derive_connectivity" came from, but it
+    #   throws errs when used in conjunction w/Soong-jin's ANAIOPAD shenanigans
+    # add_io_fillers -cells "$ioFillerCells" -logic -derive_connectivity
+    # add_io_fillers -cells "$ioFillerCells" -logic
+    addIoFiller -cell "$ioFillerCells" -logic
+
+    ########################################################################
+    # Note this section has nothing to do with IO fillers.
+    # This could easily be a separate script, call it something like "rte_mmadness.tcl"
+
+    # [stevo]: connect corner cells to RTE
+    # delete and recreate cell to set "is_physical" attribute to false (can't connect net to pin of physical-only cell)
+    #     
+    # FIXME steveri 1912 - "get_db insts pads/corner*" does
+    # nothing. Did you mean to say "get_db insts corner*"?
+    # NOTE THAT elsewhere (floorplan.tcl maybe) we *disconnected* all the RTE pins
+    foreach inst [get_db insts pads/corner*] {
+      puts "this line is never reached :("
+      set loc [get_db $inst .location]
+      set ori [get_db $inst .orient]
+      set name [get_db $inst .name]
+      delete_inst -inst $name
+      create_inst -inst $name -cell PCORNER  -status fixed \
+          -location [lindex $loc 0] -orient [string toupper $ori]
+      # connect_pin -net pads/rte -pin RTE -inst $name
+      # connect_pin -inst $name -pin RTE -net pads/rte
+        attachTerm $name RTE pads/rte
+    }
+    # At this point there are no rte or esd nets (see above)
+    # (e.g. 'get_db nets rte' returns empty set)
+    # so this does nothing as well
+    set_db [get_db nets rte] .skip_routing true
+    set_db [get_db nets rte] .dont_touch true
+    set_db [get_db nets esd] .skip_routing true
+    set_db [get_db nets esd] .dont_touch true
+
+    # end of "rte_madness.tcl"
+    ########################################################################
+
+
+    # [sr 1912] Re-create (but don't place) corner_ur that was deleted just above
+    # This prevents missing-cell error when/if read results_syn again, e.g.
+    # 
+    # **ERROR: (TCLCMD-917): Cannot find 'cells' that match 'corner_ur'
+    # (File /sim/steveri/garnet/tapeout_16/synth/GarnetSOC_pad_frame/
+    # powerplanned.db/libs/mmmc/syn_out._default_constraint_mode_.sdc,
+    # Line 105657): "set_dont_touch [get_cells corner_ur]"
+    create_inst -inst corner_ur -cell PCORNER
+
+    # Done!
+    # snap_floorplan -all; check_floorplan
+    snapFPlan -all
+    checkFPlan
+
+# Uh, now stream it out i guess
```

Comment:
Yeah this script shouldn't be concerned with streaming out the GDS.

Reply:
How about I fix this in the next pull (should be soon)?